packages/core/admin/admin/src/content-manager/pages/EditSettingsView/utils/tests/layout.test.js

```diff
@@ -100,7 +100,7 @@ describe('Content Manager | containers | EditSettingsView | utils | layout', ()
   describe('getDefaultInputSize', () => {
     it('Should return 6 if the type is unknown, undefined or text', () => {
       expect(getDefaultInputSize(undefined)).toBe(6);
-      expect(getDefaultInputSize('unkown')).toBe(6);
+      expect(getDefaultInputSize('unknown')).toBe(6);
```

Comment:
kinda scary

Reply:
This one was ok too, we could have added a name like `'test'`